src/views/feed/ActivityFeed/components/MessageThread.vue

```diff
@@ -1,12 +1,12 @@
 <template>
   <div>
     <div class="page-content card">
-      <MessageItem
+      <PrivateMessageItem
         v-for="(item, index) in messages"
         :key="item.created"
-        v-bind="item"
-        :article="article"
-        :first="index === 0"
+        :message="item"
+        :notification="notification"
+        :style="{paddingLeft: !(index === 0) && '50px'}"
```

Comment:
I'm confused by this line; If you put false does it not use the property? Also, why not `index !== 0`?
`&&` for with only one boolean is always confusing to me, so we should minimize all other complications that come alongside it IMO